setupLibs.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2017 Basis Technology.
+#
+# This software is distributed under the Common Public License 1.0
+
```

Comment:
Add a comment to the effect of "This script makes the repositories needed to compile The Sleuth Kit and its dependencies."